fuse_models/test/test_unicycle_2d_state_cost_function.cpp

```diff
@@ -99,7 +99,8 @@ TEST(CostFunction, evaluateCostFunction)
 
   // Check jacobians are correct using a gradient checker
   ceres::NumericDiffOptions numeric_diff_options;
-  ceres::GradientChecker gradient_checker(&cost_function, NULL, numeric_diff_options);
+  std::vector<const ceres::Manifold *> parameterizations;
```

Comment:
I had an issue here where my desktop was using a more recent version of Ceres where LocalParameterization has been deprecated in favor of Manifold.

Because of that using `nullptr` or `NULL` led to an ambiguous resolution of the constructor (it was overloaded to be able to take in either variant), leading me to do the explicit instantiation of the temp variable.

However, I see that [Ceres ended up fully removing LocalParameterization](https://github.com/ceres-solver/ceres-solver/commit/68c53bb39552cd4abfd6381df08638285f7386b3#diff-cfdf06c27378f4010837f18ebe34b7710a139ff4f1802158f776d4c8668a4b66L90)

I think I'll use a ceres version macro to resolve this situation..